book/src/runtime.md

```diff
@@ -1,20 +1,22 @@
 # The Runtime
 
-The runtime is a concurrent transaction processor. Transactions specify their
-data dependencies upfront and dynamic memory allocation is explicit. By
-separating program code from the state it operates on, the runtime is able to
-choreograph concurrent access. Transactions accessing only credit-only
-accounts are executed in parallel whereas transactions accessing writable
-accounts are serialized.  The runtime interacts with the program through an
-entrypoint with a well-defined interface.  The userdata stored in an account is
-an opaque type, an array of bytes. The program has full control over its
-contents.
-
-The transaction structure specifies a list of public keys and signatures for
-those keys and a sequential list of instructions that will operate over the
-states associated with the account keys.  For the transaction to be committed
-all the instructions must execute successfully; if any abort the whole
-transaction fails to commit.
+At a very high level, the state for all the accounts in solana is a simple key
```

Comment:
This set of changes doesn't seem obviously "better" to me, but a lot of the other changes do.  For starters, it's filled with typos.  Would you consider splitting this into two PRs so that we can get all the obvious improvements merged before next week's release?